scripts/generate_env_vars.sh

```diff
@@ -136,7 +136,7 @@ function gen_env_file {
 # ------------------------------------------------------------------
 # Releases
 # ==================================================================
-AETHER_VERSION=2.0.0
+AETHER_VERSION=2.0.2
```

Comment:
```suggestion
AETHER_VERSION=2.0.3
```

:thinking:

Reply:
We haven't released 2.0.3 yet have we?

Reply:
_I see the future_ :crystal_ball:

Reply:
Anyway, tests are not passing yet :disappointed_relieved: